packages/web-app-files/src/helpers/resources.js

```diff
@@ -324,9 +332,44 @@ export function buildShare(s, file, allowSharePermission) {
   if (parseInt(s.share_type) === ShareTypes.link.value) {
     return _buildLink(s)
   }
+  if (parseInt(s.share_type) === ShareTypes.space.value) {
+    return buildSpaceShare(s, file)
+  }
+
   return buildCollaboratorShare(s, file, allowSharePermission)
 }
 
+export function buildSpaceShare(s, spaceId) {
+  let permissions, role
+
+  switch (s.role) {
+    case spaceManager.inlineLabel:
```

Comment:
Please don't use display text as cases. When you rename the `name` props of the roles (see comments in role.ts) you should be able to use the respective name prop as cases.